keyboards/mmd/km75/km75.c

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2022 Jacky
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "quantum.h"
+
+
+void keyboard_pre_init_kb(void) {
+    gpio_set_pin_output(A8);
+    gpio_write_pin_high(A8);
```

Comment:
What is this for?

Reply:
usb Pull up

Reply:
Doubt.  

2 of the 3 boards are using Tri mode wireless.
http://www.i-game.tech/qmkjpxl

Reply:
Understand your concern. i-game is just design for MMD, For sure, i-game has 2 3 mode designs for other brand under developing now.(try to be rn42 compatible). but not this one.

Reply:
Whilst the code is “open” the licenses are incompatible, as I’d stated previously.

You cannot use STM32_WPAN in a GPL-licensed product.

Reply:
I put wireless code there. Although MMD KM75 did not use.